backend/prisma/schema.prisma

```diff
@@ -7,28 +7,12 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-// model Post {
-//   id        Int      @id @default(autoincrement())
-//   createdAt DateTime @default(now())
-//   updatedAt DateTime @updatedAt
-//   title     String   @db.VarChar(255)
-//   content   String?
-//   published Boolean  @default(false)
-//   author    User     @relation(fields: [authorId], references: [id])
-//   authorId  Int
-// }
-
-// model Profile {
-//   id     Int     @id @default(autoincrement())
-//   bio    String?
-//   user   User    @relation(fields: [userId], references: [id])
-//   userId Int     @unique
-// }
+model User {
+  id   String @id @default(cuid())
```

Comment:
also, im curious if you had a specific reason for choosing `cuid` here

Reply:
i confused cuid with uuid, but ultimately looking into the differences i decided that uuid was the more secure choice. this is because when i read up on the cuid documentation, there was a deprecation. also prisma supports uuid from its theory whereas cuid support is dependent on external code.